AUTHORS

```diff
@@ -56,5 +56,6 @@ Till Brychcy <till.brychcy@mercateo.com>
 Victor Williams Stafusa da Silva <victorwssilva@gmail.com>
 Yonatan Sherwin <yonatansherwin@gmail.com>
 Yun Zhi Lin <yun@yunspace.com>
+Martin Panzer <postremus1996@googlemail.com>
```

Comment:
Can you move yourself to the correct line? The list is ordered by name.

Reply:
done